### ISSUE TITLE
AstroWCS: xy2sky returns RA > 360 for high declination targets

### ISSUE THREAD

**Issue by RuslanKonno** [EranOfek/AstroPack#528]
In some cases when using AstroWCS.xy2sky the function produces RA values higher than 360, this seems a common issue for high declination targets, like the LAST fields 1781 and 1782. One can thus find sources with too high RA in catalogs for these fields, e.g.

```
>> TestAC = AstroCatalog('/marvin/LAST.01.06.04/2024/07/12/proc/210043v0/LAST.01.06.04_20240712.210033.046_clear_1781_000_001_017_sci_coadd_Cat_1.fits')
>> sum(TestAC.getCol('RA') > 360)

ans =

    83
```

**Comment by EranOfek** [comment]
can you please verify that the coordinates are correct (even though they
are >360)?
If this is just a range problem then the fix will be easy.
Yossi is cc'ed.

**Comment by RuslanKonno** [comment]
The example above has sources with RA values in range of [99, 457], feels like some trigonometric stability issue around Dec=90 deg.

**Comment by EranOfek** [comment]
The question is: after you convert the RA range into 0 to 360, is the
coordinates correct (e.g., if you match them to GAIA
using imProc.match.match_catsHTM) - is this correct?

**Comment by RuslanKonno** [comment]
Right, if I subtract the minimum from all values, then the range is exactly [0, 360] degrees. If I then pick out some bright sources and compare against simbad or GAIADR3 on Vizier, there's no match of a similarly bright objects within 1 arcmin. 

Not sure if just matching imProc.match.match_catsHTM helps, I wouldn't know if it's matching to the correct objects. 

Also it seems that adjusting the RA by just subtracting the minimum gives coordinates outside of the image.

**Comment by RuslanKonno** [comment]
The issue persists and gives an incorrect coordinate estimation for reported transients
http://ap-ag-ofek-pc/candidates/3627/?filter_value=all&start_datetime=2025-09-17%2014%3A36%3A00&end_datetime=2025-09-18T21%3A09
